spec/spec_helper.rb

```diff
@@ -152,4 +152,16 @@
     # Times out after 10 minutes
     Timeout.timeout(600) { example.run }
   end
+
+  config.around(:example, :without_config_file) do |example|
+    CommandHelpers.unset_config_file
```

Comment:
This doesn't delete the tempfile created by `configure_config_file`. Maybe we should just call `delete_config_file` here and set `ENV["CONFIG_FILE_PATH"]` to `nil` in there, without the need for `unset_config_file`.

Reply:
Since we're not using `unset_config_file`, let's remove it and add `ENV["CONFIG_FILE_PATH"] = nil` to `delete_config_file`.

Reply:
Done, also added `@@tmp_config_file = nil` because this variable should be nil if config file is deleted